lib/board.rb

```diff
@@ -0,0 +1,55 @@
+require 'pry'
+class Board
+  attr_accessor :cells
+  attr_reader   :player
+  # def cells
+  #   []
+  # end
+
+  def initialize
+    @cells = Array.new(9," ")
+  end
+
+  def reset!
+    @cells = Array.new(9, " ")
+  end
+
+  def display
+    puts " #{@cells[0].upcase} | #{@cells[1].upcase} | #{@cells[2].upcase} "
+    puts "-----------"
+    puts " #{@cells[3].upcase} | #{@cells[4].upcase} | #{@cells[5].upcase} "
+    puts "-----------"
+    puts " #{@cells[6].upcase} | #{@cells[7].upcase} | #{@cells[8].upcase} "
+
+  end
+
+  def position(pos)
+    @cells[pos.to_i - 1]
+  end
+
+
+  def full?
+    @cells.include?(" ") ? false : true
+  end
+
```

Comment:
For this and several other methods, consider using Ruby's implicit return rather than a ternary.  In this example, `@cells.include?(" ")` will return true or false, so it alone would be sufficient